framework/src/base/Assembly.C

```diff
@@ -515,6 +515,10 @@ Assembly::createQRules(QuadratureType type, Order order, Order volume_order, Ord
   for (unsigned int dim = 0; dim <= _mesh_dimension; dim++)
     _holder_qrule_face[dim] = QBase::build(type, dim - 1, face_order).release();
 
+  _holder_qrule_fv_face.resize(4);
```

Comment:
What’s with this hard-codes size here but then you use the mesh dimension immediately below?

Reply:
fixed